FlagsmithClient/src/main/java/com/flagsmith/Flagsmith.kt

```diff
@@ -101,6 +165,20 @@ class Flagsmith constructor(
             analytics?.trackEvent(featureId)
             foundFlag
         })
+    }.also { lastUsedIdentity = identity }
+
+    private fun startRealTimeListener() {
+        if (eventService == null) return
+        eventService.sseEventsFlow.onEach {
+            getFeatureFlags { res ->
+                if (res.isFailure) {
+                    Log.e("Flagsmith", "Error getting flags in SSE stream: ${res.exceptionOrNull()}")
+                    return@getFeatureFlags
+                }
+            }
+        }.catch {
+            Log.e("Flagsmith", "Error in SSE stream: $it")
+        }
     }
```

Comment:
This seems to redefine some of the logic that is defined when creating the FlagsmithEventService [here](https://github.com/Flagsmith/flagsmith-kotlin-android-client/pull/28/files#diff-f2829a622fac22bed7faa6da9c600bf86215ba6231def1f5119990c54a31d095R57-R80). Is there a reason we're having to define this twice? Can we simplify this at all?

Reply:
I think you're right and might be left over from development. Tests pass without this so I think we're fine without it.